manifests/init.pp

```diff
@@ -132,6 +132,9 @@
 #   available, likely results in performance degradation due to I/O blocking and is not recommended in most cases. Modifying this parameter should
 #   be done incrementally with benchmarking at each step to determine an optimal value for your deployment.
 #
+# @param worker_ttl
+#   The number of seconds before a worker should be considered lost.
```

Comment:
Can you clarify here that this applies to the _pulpcore workers_ for the tasking system, not the content or API gunicorn workers? Since that is likely to be confusing to many/most users.

It would be great to offer some guidance as well regarding tuning this parameter and in what circumstances might that need arise (during large content migrations? other workloads as well?). Can this be disabled by setting it to 0? What are the advantages and drawbacks?

Reply:
> Can you clarify here that this applies to the _pulpcore workers_ for the tasking system, not the content or API gunicorn workers? Since that is likely to be confusing to many/most users.

I can add a bit more info. This text was taken directly from Pulp.



> It would be great to offer some guidance as well regarding tuning this parameter and in what circumstances might that need arise (during large content migrations? other workloads as well?). Can this be disabled by setting it to 0? What are the advantages and drawbacks?

It would be great to have that, but since we don't know that I would opt not to include anything related to it for now.